src/utils/utils.ts

```diff
@@ -461,7 +461,7 @@ export const orderToJSON = (order: Order): OrderJSON => {
     staticExtradata: order.staticExtradata,
     paymentToken: order.paymentToken.toLowerCase(),
     quantity: order.quantity.toString(),
-    basePrice: order.basePrice.toString(),
+    basePrice: makeBigNumber(order.basePrice),
```

Comment:
how did this typecheck? the orderToJSON method needs to output valid JSON, which needs to convert bignumbers to strings.